src/App.js

```diff
@@ -1,16 +1,154 @@
-import './App.css';
-import { useState } from 'react'
+import "./styles/App.css";
+import { useState } from "react";
+import Map from "./components/Map";
+import leafletPip from "leaflet-pip";
+import { L, GeoJson } from "leaflet";
+import Modal from "./components/Modal";
 
-import Map from './components/Map'
+//----------------Initializing Lat/Long variables and score variable-----------------///
+let randLat = randLatGen(45.007561302382754, 42.730315121762715);
+let randLon = randLonGen(-73.26, -71.28);
+let playerScore = 100;
+let startCenter = [randLat, randLon];
+
+
+//---------Functions to generate lat and long of random point-----//
+function randLonGen(min, max) {
+  let randLon = Math.random() * (max - min) + min;
+  return randLon;
+}
+
+function randLatGen(min, max) {
+  let randLat = Math.random() * (max - min) + min;
+  return randLat;
+}
+
+//--------------------deducts players point by the passed argument-----------//
+function playerDeductPoint(points) {
+  let newScore = playerScore - points;
+  playerScore = newScore;
+  console.log(newScore);
+}
+
+//----------------------------Main App Function---------------------------------//
 
 function App() {
+  const [center, setCenter] = useState([43.88, -72.7317]);
+  const [zoom, setZoom] = useState(8);
+
+  
+//-----Function to start game, enable/disable buttons, and zoom in to random point---//
+  function startGame() {
+   // toggleDisable(); Working on this function
+    setZoom(18);
+    setCenter(startCenter);
+  }
+//-------------Function to disable/enables when game starts. Working on disabling directional buttons---//
+  function toggleDisable() {
+    document.getElementById("start-button").disabled = true;
+    document.getElementById("guess").disabled = false;
+    document.getElementById("quit").disabled = false;
+    // document.getElementById('East').disabled= false;
+    // document.getElementById('West').disabled= false;
+    // document.getElementById('North').disabled = false;
+    // document.getElementById('South').disabled = false;
+    // document.getElementById('Return').disabled = false;
```

Comment:
Along with not changing the HTML file, we don't want to ever target any elements using `document.getElementById` and change their properties this way. Because of using JSX and React we can add the disabled property to the element directly and set the value to a variable like so `<button id="start-button" disabled={variable}>Start</button>` and now we can target that variable to be either true or false by adjusting it in your `toggleDisable` function